lib/pwsh/util.rb

```diff
@@ -49,8 +49,12 @@ def snake_case(string)
     #
     # @return [Hash] Hash with all keys snake_cased
     def snake_case_hash_keys(hash)
```

Comment:
nit: Not really a hash now is it

Reply:
🤔 would `object` be the better name here, then?